Example/Pods/Expecta+Snapshots/EXPMatchers+FBSnapshotTest.h

```diff
@@ -0,0 +1,14 @@
+#import <Expecta/Expecta.h>
```

Comment:
hmmmm no. Ill look into changing that.

On Tue, May 3, 2016 at 10:06 AM, managedapps <notifications@github.com>
wrote:

> In Example/Pods/Expecta+Snapshots/EXPMatchers+FBSnapshotTest.h
> <https://github.com/managedapps/ManagedStatsOC/pull/20#discussion_r61886370>
> :
>
> > @@ -0,0 +1,14 @@
> > +#import <Expecta/Expecta.h>
>
> Any reason you checked in a cocoapod?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/managedapps/ManagedStatsOC/pull/20/files/44db1bfa1b200e1e3452f599e0b60fe5b7acecff..58effb93e602e6bfe9cec57553b6c208eece2e86#r61886370>
>